build_test/test/test_builder_test.dart

```diff
@@ -169,13 +169,15 @@ void main() {
           '.dart': ['.fake']
         },
         build: (step, _) async {
-          var lib = await step.inputLibrary;
-          var errors = await lib.session.getErrors(lib.source.fullName);
-          expect(
-              errors.errors.map((e) => e.message),
-              contains(contains(
-                  'This requires the \'extension-methods\' language feature to be '
-                  'enabled.')));
+          try {
+            await step.inputLibrary;
+          } on SyntaxErrorInAssetException catch (e) {
+            expect(
```

Comment:
Not sure if I missed it in a previous review or if this is new, this expectation won't work if it stops throwing.

```dart
expect(step.inputLibrary, throwsA(isA<SyntaxErrorInAssetException>().having(...
```

Reply:
Oh right, ill update that